src/components/AppHeader/styled.tsx

```diff
@@ -0,0 +1,36 @@
+// @ts-ignore
+import styled from 'styled-components';
+
+const Nav = styled.div`
+  overflow: hidden;
+  background-color: #333;
+  position: fixed; 
+  top: 0;
+  width: 100%;
+  a {
+    float: left;
+    display: block;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+  }
+  a:hover {
```

Comment:
You usually want to apply the same styles to `:hover`, `:active` and `:focus`, to increase accessibility and make life easier to keyboard users 👌

Reply:
I didn't know about that. Fixed. Thanks!